c7n/actions.py

```diff
@@ -390,6 +390,8 @@ class Notify(EventAction):
               - event-user
               - resource-creator
               - email@address
+             to_if_no_owner:
```

Comment:
I dig the functionality, and lgtm,  just wondering if its a little odd in the spelling... what would you think about something without the verbs , perhaps an adjective ala `owner_absent_contact` or `owner_missing_contact`

Reply:
Yeah, I suppose it is a bit awkward. ``owner_absent_contact`` sounds good to me. I can fix this up and re-push in the morning.